tests/unit_and_functional/conftest.py

```diff
@@ -61,11 +72,12 @@ def setup_bodywork_test_project(
             capture_output=True, encoding='utf-8')
         os.mkdir(bodywork_output_dir)
         yield True
-    except CalledProcessError as e:
+    except Exception as e:
         raise RuntimeError(f'Cannot create test project Git repo - {e.output}.')
     finally:
         # TEARDOWN
         shutil.rmtree('{}/.git'.format(project_repo_location), onerror=on_error)
+        shutil.rmtree('{}/.git'.format(cloned_project_repo_location), ignore_errors=True, onerror=on_error)
```

Comment:
Could we stick to using f-strings - i.e. `f'{cloned_project_repo_location}/.git'`. I believe that the `.format` approach is slowly being phased-out. I failed to notice this in the above line when I reviewed it the first time around.

Reply:
Yeah this shouldn't be different from the rest.